godot-core/src/builtin/callable.rs

```diff
@@ -392,10 +393,21 @@ mod custom_callable {
     ) {
         let arg_refs: &[&Variant] = Variant::borrow_ref_slice(p_args, p_argument_count as usize);
 
-        let c: &mut C = CallableUserdata::inner_from_raw(callable_userdata);
+        let name = {
+            let c: &C = CallableUserdata::inner_from_raw(callable_userdata);
+            c.to_string()
+        };
+        let ctx = CallContext {
+            class_name: Cow::Borrowed("<Callable>"),
+            function_name: name.as_str(),
+        };
```

Comment:
Maybe consider adding a constructor, so this can be invoked as:
```suggestion
        let ctx = CallContext::custom_callable(name.as_str());
```